source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -324,6 +303,11 @@ void BioXASAppController::setupUserInterface()
 	addPersistentView(new BioXASPersistentView());
 }
 
+void BioXASAppController::setupBeamlineSignalConnections()
```

Comment:
I defined this function as a pure virtual function in CLSBeamline. Therefore, all the subclass must implement it. The older beamlines are implementing in this way --- organize all the signals as a code block. The BioXAS beamlines and new SGM beamline are NOT doing in this way. 

I don't think it hurts to leave it here. Maybe we will change minds in the future